core/src/main/java/org/apache/carbondata/core/metadata/schema/PartitionInfo.java

```diff
@@ -43,13 +44,30 @@
   private List<List<String>> listInfo;
 
   /**
-   * number of partitions
+   * number of hash partitions
    */
-  private int numPartitions;
+  private int hashNumber;
+
+  /**
+   * total count of partitions
+   */
+  private int numberOfPartitions;
```

Comment:
I think it represent different concept. Though they are equal in hash table. That would be more clear for developers.